### PR TITLE
Hello, Github actions and good-bye Travis.

### PR DESCRIPTION
**Pull request by shkumagai**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Test
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```
Comment (tk0miya) [inline .github/workflows/test.yaml:9]: Could you add 3.10-dev also here? In sphinx-doc/sphinx, it has been installed via deadsnakes/action.
https://github.com/sphinx-doc/sphinx/blob/3.x/.github/workflows/main.yml#L30-L33
https://github.com/sphinx-doc/sphinx/blob/3.x/.github/workflows/main.yml#L44-L48

File: tox.ini [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+skip_missing_interpreters = true
 envlist=flake8,mypy,py35,py36,py37,py38,py39
 
+[gh-actions]
+python =
+    3.5: py35
+    3.6: py36
+    3.7: py37
+    3.8: py38
+    3.9: py39
```
Comment (tk0miya) [inline tox.ini:11]: How cool! I've never knew such a good plug-in :-)

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: pycache
+        uses: actions/cache@v2
+        with:
+          path: .venv
+          key: python-${{ github.sha }}
```
Comment (tk0miya) [inline .github/workflows/test.yaml:27]: Is this working fine? It seems nobody uses `.venv`. Is it automatically used? Do you intend to cache `.cache/pip` instead?
https://github.com/actions/cache/blob/main/examples.md#python---pip

**Review by tk0miya — CHANGES_REQUESTED** [review]
Thank you for your contribution! It must be helpful to us :-)
LGTM with nits.
Could you check my comments, please?

File: .github/workflows/test.yaml (hunk 3 above)
Comment (shkumagai) [reply .github/workflows/test.yaml]: Ooops, I did miss it.

File: .github/workflows/test.yaml (hunk 1 above)
Comment (shkumagai) [reply .github/workflows/test.yaml]: I added py3.10-dev to the target versions list.
It seems to work fine ;-)

File: .github/workflows/test.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        uses: actions/setup-python@v2
+        if: "!endsWith(matrix.python-version, '-dev')"
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup python {{ matrix.python-version }} (via deadsnakes)
+        uses: deadsnakes/action@v2.1.1
+        if: "endsWith(matrix.python-version, '-dev')"
+        with:
+          python-version: ${{ matrix.python-version }}
```
Comment (shkumagai) [inline .github/workflows/test.yaml:41]: Add python nightly build version via the deadsnakes plugin.

**Comment by shkumagai** [comment]
@tk0miya I've fixed all you requested points.
If you have any other requests, let me know them.

File: .github/workflows/test.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: pycache
+        uses: actions/cache@v2
+        with:
+          path: .cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
```
Comment (tk0miya) [inline .github/workflows/test.yaml:27]: All dependency packages are defined at setup.py. How about specifying it instead?

**Comment by tk0miya** [comment]
Thank you for your great work! Merging now :-)

**Pull request merged**